corehq/motech/repeaters/models.py

```diff
@@ -409,7 +415,7 @@ def register(self, payload, fire_synchronously=False):
             # Prime the cache to prevent unnecessary lookup. Only do this for synchronous repeaters
             # to prevent serializing the repeater in the celery task payload
             RepeatRecord.repeater.fget.get_cache(repeat_record)[()] = self
-
+        print("\n\n\n\n-------------------- Attempt to forward -----------------\n\n\n")
```

Comment:
It seems you've managed to resolve the issue with the failing test, so just noting that this debug log should be removed.

Reply:
Ah thanks!